ubuntu_bug_triage/__main__.py

```diff
@@ -47,6 +47,14 @@ def parse_args():
         '--include-project', '-p', action='store_true',
         help='include project bugs in output'
     )
+    parser.add_argument(
+        '--status', '-s', action='append', default=[], metavar='STATUS',
+        choices=["New", "Incomplete", "Opinion", "Invalid", "Won't Fix",
+                 "Expired", "Confirmed", "Triaged", "In Progress",
+                 "Fix Committed", "Fix Released", "Incomplete (with response)",
+                 "Incomplete (without response)"],
```

Comment:
I don't think our defaults should include everything; I would prefer our defaults do exactly what we want.

Reply:
I now set `args.status` to `[]` by default, and if it stays so I set it to `ACTIONABLE_BUG_STATUSES` (our defaults) below, within `launch()`.

The reason I don't use the argparse `default` keyword directly is the fact that argparse can only append to the default value, not override it, so there would be no way to *remove* any of the default bug statuses.